MapboxNavigation/NavigationMapView.swift

```diff
@@ -342,9 +344,10 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             let function: CAMediaTimingFunction? = animated ? CAMediaTimingFunction(name: kCAMediaTimingFunctionLinear) : nil
             setCamera(newCamera, withDuration: duration, animationTimingFunction: function, edgePadding: padding, completionHandler: nil)
         } else {
-            UIView.animate(withDuration: duration, delay: 0, options: [.curveLinear, .beginFromCurrentState], animations: {
+            // While animating to overhead mode, don't animate the puck.
+            UIView.animate(withDuration: isAnimatingToOverheadMode ? 0 : duration, delay: 0, options: [.curveLinear, .beginFromCurrentState], animations: {
```

Comment:
This is fine, but it might be cleaner to have `isAnimatingToOverheadMode` affect `duration` up above:

```swift
let duration: TimeInterval = animated && !isAnimatingToOverheadMode ? 1 : 0
```

Reply:
I think it will always be false here, but ok.